android/build.gradle

```diff
@@ -2,22 +2,22 @@ group 'com.logger.logging_to_logcat'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.8.10'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
Maybe 8+?

Reply:
The version of gradle has been updated to 8.4.0 [005c6e7](https://github.com/ChristianKleineidam/logging_to_logcat/pull/6/commits/005c6e7997248c5c1e0ebde5f091de3c4af07c14)